model/android/resources.yaml

```
@@ -1,6 +1,6 @@
 groups:
   - id: resource.android
```

Should IDs and file names also be renamed for consistency?
To be done in a later PR?
Eg. `model/android/resources.yaml` → `model/android/entities.yaml`
```suggestion
  - id: entity.android
```

---

Yeah I was thinking of doing this later.  I can do this now if we think we'd like to bump that consistency now too.

Id's are almost invisible to me.

---

For me it's fine to do it later.
I was just wondering whether it was considered and if anyone else had an opinion on it.

---

if someone feels strongly, I'll update it now.

---

I think we should change it here, or at least follow soon with another PR to change it. Unless there's certain tooling that depends on these specific ids. At the very least it's one less thing in the yaml that could confuse people.

---

I think it needs to also rename resources.yaml to entities.yaml

---

Going to merge this and rename in follow-up.